README.md

```diff
@@ -34,6 +34,51 @@ To run the test go into the build directory and use:
 ./run_test.sh 
 ```
 
+
+
+## (Pilot) Build Hustle with C++20
```

Comment:
Can we the Pilot section towards the end of the readme section?

Reply:
Got it. Fixed!